recipe/meta.yaml

```
@@ -26,7 +26,9 @@ requirements:
     - setuptools_scm
   run:
     - python
-    - 'adc[channel=scimma]'
+    - adc >=0.0.3
+    - xmltodict >=0.9.0
+    - dataclasses
```

We'll need to add a specifier here that says it's only needed for python 3.6. A guide to these are in here: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors

```suggestion
    - dataclasses # [py==36]
```